internal/config/config.go

```diff
@@ -43,7 +43,11 @@ type Config struct {
 	KafkaDispatcherTopic    string
 	KafkaGroupID            string
 	KafkaInventoryTopic     string
+	KafkaPassword			string
+	KafkaSASLMechanism		string
```

Comment:
Where are these config options used? I see when the producer and consumer is created, it uses the `KafkaUsername` and `KafkaPassword`, but `KafkaSASLMechanism` and `KafkaSecurityProtocol` aren't used. Is basic auth the only supported authentication mechanism at this time?

Reply:
Yeah I think the only auth types that work with segmentio are scram and plain, and so far I have just been setting it up to work with plain(let me know if it would be appropriate to add scram as a choice as well). I think I will remove those two unused vars unless you would like me to add scram as well, in which case I will probably keep the SASLmechanism so that I can use that variable decide between plain or scram